### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by jeyaramashok**
#### What this PR does / why we need it:

Add psp, clusterrole and rolebinding guidelines

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: stable/logdna-agent/README.md [hunk 1]
```diff
@@ -19,7 +19,111 @@ This chart deploys LogDNA collector agents to all nodes in your cluster. Logs wi
 
 ### Pod Security Policies
 
-The namespace you deploy into must allow the container to run as root, and must be able to access hostPath volumes. 
+LogDNA-agent containers needs to mount host volumes to read the logs for ingestion. Therefore, this chart requires a PodSecurityPolicy that permits host volume access to be bound to the target namespace prior to installation. If the default PodSecurityPolicy on your cluster is not restrictive then this step is not needed.
```
Comment (jchunkins) [inline stable/logdna-agent/README.md]:
```suggestion
LogDNA-agent containers needs to mount host volumes to read the logs for ingestion, and needs to run as root. This chart requires a PodSecurityPolicy (with host volume and root access) to be bound to the target namespace prior to installation. If the `default` PodSecurityPolicy on your namespace is not restrictive then this step is not needed.
```

**Comment by jeyaramashok** [comment]
I used these two charts as references to come up with section on PSP

- https://github.com/IBM/charts/blob/master/stable/ibm-mobilefoundation-dev/README.md#podsecuritypolicy-requirements
- https://github.com/IBM/charts/tree/master/community/openwhisk#podsecuritypolicy-requirements

File: stable/logdna-agent/README.md [hunk 2]
```diff
@@ -19,7 +19,111 @@ This chart deploys LogDNA collector agents to all nodes in your cluster. Logs wi
...
+
+If the default is restrictive, In ICP you can create a new namespace with either a predefined PodSecurityPolicy.
+
+- Predefined PodSecurityPolicy name: [ibm-anyuid-hostpath-psp](https://github.com/IBM/cloud-pak/blob/master/spec/security/psp/README.md)
```
Comment (jchunkins) [inline stable/logdna-agent/README.md]:
```suggestion
```

File: stable/logdna-agent/README.md [hunk 3]
```diff
@@ -19,7 +19,111 @@ This chart deploys LogDNA collector agents to all nodes in your cluster. Logs wi
...
+    rule: RunAsAny
+seLinux:
+    rule: RunAsAny
+supplementalGroups:
+    rule: RunAsAny
+volumes:
+- '*'
+```
+
+- Create a [ClusterRole](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings)
```
Comment (jchunkins) [inline stable/logdna-agent/README.md]:
```suggestion
- Create a [ClusterRole](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#default-roles-and-role-bindings). Save the following contents into a file (e.g. psp.yaml)
```

File: stable/logdna-agent/README.md [hunk 4]
```diff
@@ -19,7 +19,111 @@ This chart deploys LogDNA collector agents to all nodes in your cluster. Logs wi
...
+    resources:
+    - pods
+    - secrets
+    - jobs
+    verbs:
+    - get
+    - create
+    - delete
+```
+
```
Comment (jchunkins) [inline stable/logdna-agent/README.md]:
```suggestion
kubectl command:
```
Jey... put this command here in markdown triple tilde quotes:

kubectl create -f psp.yaml

File: stable/logdna-agent/README.md [hunk 5]
```diff
@@ -19,7 +19,111 @@ This chart deploys LogDNA collector agents to all nodes in your cluster. Logs wi
...
+- Create ClusterRoleBinding
+
+Create a ClusterRoleBinding which binds ClusterRole created in previous step to default service account of the namespace.
+
+kubectl command:
```
Comment (jchunkins) [inline stable/logdna-agent/README.md]:
```suggestion
kubectl command (replace `<namespace>` with your namespace):
```

File: stable/logdna-agent/README.md [hunk 6]
```diff
@@ -19,7 +19,111 @@ This chart deploys LogDNA collector agents to all nodes in your cluster. Logs wi
...
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: logdna-clusterrole
+subjects:
+    - kind: ServiceAccount
+    name: default
+    namespace: <namespace>
+```
+
+Alternatively, a cluster admin can create a [service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) for LogDNA itself and bind the ClusterRole to it.
```
Comment (jchunkins) [inline stable/logdna-agent/README.md]:
```suggestion
Alternatively, a cluster admin can create a [service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) for LogDNA itself and bind the ClusterRole to it. Save the following contents into a file (e.g. sa.yaml):
```

File: stable/logdna-agent/README.md [hunk 7]
```diff
@@ -19,7 +19,111 @@ This chart deploys LogDNA collector agents to all nodes in your cluster. Logs wi
...
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: logdna-serviceaccount
+    namespace: <namespace>
+```
```
Comment (jchunkins) [inline stable/logdna-agent/README.md]:
```suggestion
kubectl command:
```

Jey... put this command here in markdown triple tilde quotes:

kubectl create -f sa.yaml

**Comment by jeyaramashok** [comment]
@jchunkins I have fixed the code review suggestions, pls take a look when you get a chance.

**Pull request merged**